### PR TITLE
exported RegistrationComplete screen as component

### PR DESCRIPTION
**Pull request by Manoj1907**
Fixes #4019 .

#### Changes proposed in this Pull Request:

- Componentize RegistrationComplete

#### Screenshots / Screen Recording (if applicable)

Screenshot 2023-04-19 at 9 11 42 PM

#### To Test:

- yarn start:example

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/screens/subScreens/exportedSubScreens/RegistrationComplete.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * a new user.
+ *
+ * @param firstName first name for the new user
+ * @param lastName last name for the new user
+ * @param email email address for the new user
+ * @param organization organization that the user has joined
+ *
+ * @category Component
+ */
+export const RegistrationComplete: React.FC<
```
Comment (huayunh) [inline login-workflow/src/screens/subScreens/exportedSubScreens/RegistrationComplete.tsx:30]: `RegisterSuccessSubscreen`

File: login-workflow/src/screens/subScreens/exportedSubScreens/RegistrationComplete.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                                </Typography>
+                            }
+                        />
+
+    return (
+        showInCard ? (
+            <BrandedCardContainer>{finishState}</BrandedCardContainer>
+        ) : (
+            finishState
+        )
```
Comment (huayunh) [inline login-workflow/src/screens/subScreens/exportedSubScreens/RegistrationComplete.tsx:66]: same comment about `showInCard` here

**Review by huayunh — APPROVED** [review]
No major complaints.

**Review by surajeaton — COMMENTED** [review]
Please resolve the merge conflicts.

**Pull request merged**